### ISSUE TITLE
load failed

### ISSUE THREAD

**Issue by xiaoxianSun** [devsisters/DQN-tensorflow#64]

**Comment by xiaoxianSun** [comment]
WARNING:tensorflow:From /home/sxx/RL/DQN-tensorflow/dqn/agent.py:223: calling argmax (from tensorflow.python.ops.math_ops) with dimension is deprecated and will be removed in a future version.
Instructions for updating:
Use the `axis` argument instead
[2019-06-02 22:30:40,855] From /home/sxx/RL/DQN-tensorflow/dqn/agent.py:223: calling argmax (from tensorflow.python.ops.math_ops) with dimension is deprecated and will be removed in a future version.
Instructions for updating:
Use the `axis` argument instead
WARNING:tensorflow:From /home/sxx/.local/lib/python2.7/site-packages/tensorflow/python/util/tf_should_use.py:189: initialize_all_variables (from tensorflow.python.ops.variables) is deprecated and will be removed after 2017-03-02.
Instructions for updating:
Use `tf.global_variables_initializer` instead.
[2019-06-02 22:30:41,135] From /home/sxx/.local/lib/python2.7/site-packages/tensorflow/python/util/tf_should_use.py:189: initialize_all_variables (from tensorflow.python.ops.variables) is deprecated and will be removed after 2017-03-02.
Instructions for updating:
Use `tf.global_variables_initializer` instead.
 [*] Loading checkpoints...
 [!] Load FAILED: checkpoints/Breakout-v0/min_delta--1/max_delta-1/history_length-4/train_frequency-4/target_q_update_step-10000/double_q-False/memory_size-100000/action_repeat-4/ep_end_t-100000/dueling-False/min_reward--1.0/backend-tf/random_start-30/scale-10000/env_type-detail/learning_rate_decay_step-50000/ep_start-1.0/screen_width-84/learn_start-50000.0/cnn_format-NHWC/learning_rate-0.00025/batch_size-32/discount-0.99/max_step-50000000/max_reward-1.0/learning_rate_decay-0.96/learning_rate_minimum-0.00025/env_name-Breakout-v0/ep_end-0.1/model-m1/screen_height-84/
I am new in RL , i will appreciate if someone can help me.

**Comment by xiaotangdoupaixuexiji** [comment]
我有一样的问题，你解决了吗？

**Comment by JUZI1** [comment]
> 我有一样的问题，你解决了吗？

人家看不懂中文
